src/DI/Container.php

```diff
@@ -251,7 +253,7 @@ public function createInstance(string $class, array $args = [])
 
 	/**
 	 * Calls all methods starting with with "inject" using autowiring.
-	 * @param  object  $service
+	 * @param object $service
```

Comment:
This was ok, or not?

Reply:
yes, I can revert it if you want... or split to 2nd commit

Reply:
splitted to 2nd commit and removed some other unnecessary spaces in phpdocs

Reply:
These spaces are used intentionally and across all libraries.

Reply:
Ok, I don't know why to add unnecessary spaces, but it is your opinion. Commit reverted.